bats/tests/helpers/vm.bash

```diff
@@ -17,6 +17,21 @@ pkill_by_path() {
     fi
 }
 
+windows_cleanup() {
```

Comment:
I think this should be named `wsl_cleanup`, but I would also be fine with leaving it inside the `factory_reset` function.